src/Microsoft.DotNet.Tools.Compiler.Native/ILCompilerInvoker.cs

```diff
@@ -54,7 +46,8 @@ private void InitializeArgs(NativeCompileSettings config)
             argsList.Add($"\"{inputFilePath}\"");
             
             // System.Private.CoreLib Reference
-            var coreLibPath = Path.Combine(config.IlcPath, OSCoreLibNameMap[config.OS]);
+            String[] coreLibs = new String[] { "System.Private.CoreLib.dll", "System.Private.Corelib.dll" };
+            var coreLibPath = Path.Combine(config.IlcPath, Array.Find(coreLibs, lib => File.Exists(Path.Combine(config.IlcPath, lib))));
```

Comment:
Yes. I have a change in CoreRT to do just that, but to checkin and pass our CI, CLI should look for the updated casing instead of the current casing. But again CLI should also work as is without my changes in CoreRT getting checked in. Hence this change. Super temporary.